gutenberg.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	extends: ['plugin:@wordpress/eslint-plugin/custom', './react'],
+	globals: {
+		window: true,
+		document: true,
+		wp: 'readonly',
+	},
+	rules: {
+		'react/react-in-jsx-scope': 'off',
```

Comment:
Need to ignore `@wordpress` imports for the `import/no-unresolved` rule. We can add a regex to ignore those: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md#ignore

It's really common to use DependencyExtractionWebpackPlugin to use ES6 imports for global modules exposed by WordPress.

Reply:
cc @ryanwelcher

Reply:
This has been updated.